src/filters.cpp

```diff
@@ -210,4 +208,15 @@ NAN_METHOD(Filters::New) {
 
         info.GetReturnValue().Set(info.This());
     }
+}
+
+NAN_METHOD(Filters::layers) {
+    auto layers = Nan::New<v8::Array>();
+    std::uint32_t idx = 0;
+    auto* filters = Nan::ObjectWrap::Unwrap<Filters>(info.Holder());
+    for (auto const& lay : filters->filters) {
+        Nan::Set(layers, idx, Nan::New<v8::String>(lay.first).ToLocalChecked());
+        idx++;
```

Comment:
(I was curious, even though I don't have more context)

Nit: These 2 lines should be equivalent to

```
Nan::Set(layers, idx++, Nan::New<v8::String>(lay.first).ToLocalChecked());
```

Reply:
Thanks for the review. Yes, improved in 60cff26